docs/fields/encryption.md

```diff
@@ -59,8 +63,7 @@ Note that since this backend never decrypt the stored value it's only applicable
 
 ```python
 class Hash(ormar.Model):
-    class Meta(BaseMeta):
-        tablename = "hashes"
+    ormar_config = base_ormar_config.copy()
```

Comment:
changed table name

Reply:
you didn't fix this, should be `ormar_config = base_ormar_config.copy(tablename="hashes")`